### PR TITLE
doc(website): Hexagon blog post

### PR DESCRIPTION
**Pull request by hermanschaaf**
This adds a blog post sharing the details of how Hexagon built an infrastructure data platform using CloudQuery with AWS S3, Glue and Athena. 

I also refactored the blog author code a little bit, so we have a place to put external contributors.

### CODE REVIEW

File: website/pages/blog/how-hexagon-built-an-infrastructure-data-platform.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- Resource view for Athena: CloudQuery started out as a tool for querying cloud resources using Postgres, so initially we only supported a resources view for AWS that was compatible with Postgres. However, Athena doesn’t support views that query the information schema, so a different solution was needed. We added a Lambda function that can be run to get a list of all tables, then build or update a resources view from that. This Lambda can be triggered by a CloudWatch Event Rule when the CloudQuery sync finishes, so it’s always up-to-date.
+
+## Conclusion
+
+We’d like to thank the Hexagon CCoE team for collaborating with us to push the boundaries of what is possible with CloudQuery, being active community members and for sharing their story with everyone.
+
+If you are also interested in setting up an AWS infrastructure data lake using Glue and Athena, like Hexagon, you can follow our how-to guide: [How to Build an Infrastructure Data Lake with CloudQuery, S3, Glue and Athena](https://www.cloudquery.io/how-to-guides/how-to-load-infrastructure-data-into-athena). Or if you have any other questions or comments, we are always happy to chat on our Discord.
```
Comment (disq) [inline website/pages/blog/how-hexagon-built-an-infrastructure-data-platform.mdx:51]:
```suggestion
If you are also interested in setting up an AWS infrastructure data lake using Glue and Athena, like Hexagon, you can follow our how-to guide: [How to Load Infrastructure Data into AWS Athena](https://www.cloudquery.io/how-to-guides/how-to-load-infrastructure-data-into-athena). Or if you have any other questions or comments, we are always happy to chat on our Discord.
```

File: website/pages/blog/how-hexagon-built-an-infrastructure-data-platform.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Architecture
+
+The Hexagon CCoE team (and co-authors of this post) wanted to build a fully serverless data lake that would be able to collect data from all of Hexagon’s cloud accounts and store it in a single location (data lake). They also wanted to be able to query this data using SQL, and to be able to visualise it using their existing tools (like AWS QuickSight). Lastly, they wanted to be able to explore the history of their cloud configuration over time. To accomplish this, they built a serverless data pipeline that uses CloudQuery to collect data from all of their cloud accounts, and then stores it in S3. They then use Glue to ingest the data into Glue DB in a format that Athena can query, and finally Athena is used to query the data and visualise it using QuickSight.
```
Comment (erezrokah) [inline website/pages/blog/how-hexagon-built-an-infrastructure-data-platform.mdx:18]:
```suggestion
The Hexagon CCoE team (and co-authors of this post) wanted to build a fully serverless data lake that would be able to collect data from all of Hexagon’s cloud accounts and store it in a single location (data lake). They also wanted to be able to query this data using SQL, and to be able to visualise it using their existing tools (like AWS QuickSight). Lastly, they wanted to be able to explore the history of their cloud configuration over time. To accomplish this, they built a serverless data pipeline that uses CloudQuery to collect data from all of their cloud accounts, and then stores it in S3. They then use Glue to ingest the data into Glue DB in a format that Athena can query, and finally Athena is used to query the data and visualise it using QuickSight.
```

Can we unwrap the CCoE acronym? That should make it more clear what the team does

File: website/pages/blog/how-hexagon-built-an-infrastructure-data-platform.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+![Hexagon Architecture](/images/blog/how-hexagon-built-an-infrastructure-data-platform/hexagon-architecture.png)
+
+- In Step 1, an EventBridge cron rule is triggered on a daily basis. This event triggers the creation of two ECS tasks in an ECS Cluster (Step 2).
+- In Step 3, the AWS CLI fetches the latest plugin configs from the Master S3 Bucket and in Step 4, CloudQuery CLI, performs API calls to all child accounts and collects all the data (Step 5). This is then stored in an S3 Bucket in a dedicated account for CloudQuery. This data is in Parquet format, using the S3 destination plugin (Step 6).
+- In Step 7, a Glue Crawler is triggered on a daily basis to ingest the data from the S3 Bucket. This is then dumped into a Glue DB (Step 8). Please note each resource type has its own Glue table.
+- Step 11 starts by an EvenBridge cron rule triggered on a daily basis. This rule will then summon a Lambda function ([provided by CloudQuery](https://github.com/cloudquery/cloudquery/tree/main/plugins/source/aws/views/athena)). This function (Step 12) updates a resource view called aws_resources: a single view that contains all resources from all accounts.
```
Comment (erezrokah) [inline website/pages/blog/how-hexagon-built-an-infrastructure-data-platform.mdx:27]:
```suggestion
- Step 11 starts by an EvenBridge cron rule triggered on a daily basis. This rule will then summon a Lambda function ([provided by CloudQuery](https://github.com/cloudquery/cloudquery/tree/main/plugins/source/aws/views/athena)). This function (Step 12) updates a resource view called `aws_resources`: a single view that contains all resources from all accounts.
```

**Review by erezrokah — APPROVED** [review]
Looks great, nothing blocking 🚀

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**